data/ecosystems/b/bnb-chain-bsc.toml

```diff
@@ -2273,9 +2273,6 @@ missing = true
 [[repo]]
 url = "https://github.com/AhmedSalikRiaz/NFT_marketplace_using_React_and_Moralis"
 
-[[repo]]
-url = "https://github.com/ahmetenesdur/chat-dapp"
```

Comment:
why delete this URL?

Reply:
We noticed some duplicate repos and removed them